artifactory/commands/generic/delete.go

```diff
@@ -59,7 +59,7 @@ func (dc *DeleteCommand) GetPathsToDelete() (*content.ContentReader, error) {
 	if errorutils.CheckError(err) != nil {
 		return nil, err
 	}
-	servicesManager, err := utils.CreateServiceManager(serverDetails, dc.retries, dc.DryRun())
+	servicesManager, err := utils.CreateServiceManager(serverDetails, dc.retries, dc.retryWaitTime, dc.DryRun())
```

Comment:
retryWaitTime --> retryWaitMilliSecs

Let's make this change all across the code, including methods like ```SetHttpRetryWaitTime```.